aten/src/ATen/native/cuda/ForeachFunctors.cuh

```diff
@@ -547,5 +548,19 @@ struct TernaryOpScalarFunctor {
   }
 };
 
+template <typename T>
+struct power_functor {
+  C10_DEVICE T operator()(const T& a, const T& b) const {
+    return pow(a, b);
+  }
+};
+
+template <typename T>
+struct alt_power_functor {
```

Comment:
Why do you need this one?

Reply:
`multi_tensor_apply` assumes that `TensorList` is SelfArgument but `_foreach_pow(scalar, tensor_list)` doesn't satisfy the assumption. so I needed to exchange the order of arguments before passing them to `pow` function

Reply:
Can you make the name more explicit then? `reverse_power_functor` would match better the usual `rmethod` that exist in python world.